documentation/general/dash-sonic-hld.md

```diff
@@ -221,6 +229,8 @@ qos                      = Associated Qos profile
 underlay_ip              = PA address for Inbound encapsulation to VM
 admin_state              = Enabled after all configurations are applied. 
 vnet                     = Vnet that ENI belongs to
+pl_sip_encoding          = Private Link encoding for IPv6 SIP transpositions; Format "field:<bit_offset>:<size_in_bits>:<value in hex>:field:<bit_offset>:<size_in_bits>:<value in hex>"
```

Comment:
This could potentially take up to the first 96 bits of the IPv6 SIP right? So we would use this to calculate the prefix on the fly, instead of using a provided prefix like in ST. What should the default be if not all 96 bits of the prefix are accounted for?

Reply:
If its not provided, it should be from mapping table. The example section has the details